code/modules/mob/living/basic/pets/pet.dm

```diff
@@ -17,7 +17,19 @@
 /mob/living/basic/pet/Initialize(mapload)
 	. = ..()
 
-	AddElement(/datum/element/atmos_requirements, list("min_oxy" = 5, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0), 1)
+	// String assoc list returns a cached list, so this is like a static list to pass into the element below.
+	var/list/habitable_atmos = string_assoc_list(list(
```

Comment:
Why not use a static list directly then? This is a pattern we do a lot, such as with contextual screentips.

Reply:
We could. Stickmen could too (the one other use of this element).

Reply:
Oh right, because it's a string assoc list, so the value returned is a static global anyway.

Reply:
But then why not use static directly and avoid the indirection altogether?

Reply:
From what I would intuit, `string_assoc_list` is going to be most useful for things like dynamic lists, where we can't just use `static` but we still want them cached. We don't need that here.